.github/workflows/cd.yml

```diff
@@ -0,0 +1,118 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build_and_upload_artifact_for_macos_and_linux:
+    strategy:
+      matrix:
+        config:
+        - { os: ubuntu-latest, name: 'linux'}
+        - { os: macos-latest, name: 'macos'}
+    runs-on: ${{ matrix.config.os }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Fetch tag git metadata
+        run: git fetch --force --update-shallow --depth 1 origin 'refs/tags/*:refs/tags/*'
+      - uses: actions/setup-dotnet@v1.8.1
+        with:
+          dotnet-version: 5.0.x
+      - name: Configure cmake
+        run: |
+          mkdir cmake-build-release && cd cmake-build-release && cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release ..
+      - name: Build bufc
+        run: |
+          make bufc
+        working-directory: cmake-build-release
+      - name: Update Nuget Source
+        run: |
+          dotnet nuget update source GitHub-StirlingLabs --source https://nuget.pkg.github.com/StirlingLabs/index.json --username ${{github.repository_owner}}  --password ${{github.token}} --store-password-in-clear-text
+        working-directory: net
+      - name: Build BigBuffers Net Projects
+        run: |
+          dotnet build BigBuffers.sln -c Release
+        working-directory: net
+      - name: upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: bufc-${{ matrix.config.name }}
+          path: cmake-build-release/bufc
+  build_and_upload_artifact_for_windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+      - name: Fetch tag git metadata
+        run: git fetch --force --update-shallow --depth 1 origin 'refs/tags/*:refs/tags/*'
+      - uses: actions/setup-dotnet@v1.8.1
+        with:
+          dotnet-version: 5.0.x
+      - name: Configure cmake
+        run: |
+          mkdir cmake-build-release && cd cmake-build-release && cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
When building locally with MSys2 Makefiles (or Ninja under Msys2's CMake), there's no release subdirectory.

Visual Studio CMake targets are de facto multi-configuration environments, so they have Debug and Release output directories.

You'd have to add `-DCMAKE_RUNTIME_OUTPUT_DIRECTORY=${{github.workspace}}/cmake-build-release` to deal with that correctly.

It should be fine to leave it as is, but note the change in the project to add a search location instead of changing the search location.